src/components/molecules/email-form.tsx

```diff
@@ -0,0 +1,102 @@
+import { useState } from 'react'
+
+import { Button } from '@/components/atoms/button.tsx'
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogFooter,
+  DialogHeader,
+  DialogTitle,
+} from '@/components/atoms/dialog.tsx'
+import { Popover, PopoverContent, PopoverTrigger } from '@/components/atoms/popover.tsx'
+import { RichTextEditor } from '@/components/atoms/rich-text-editor.tsx'
+import { Typography } from '@/components/atoms/typography.tsx'
+import { Vendor } from '@/schemas/vendor.ts'
+
+interface EmailFormProps {
+  selectedVendors: Vendor[]
+  onClose: () => void
+  defaultContent?: string
+}
+
+export function EmailForm({ selectedVendors, onClose, defaultContent = '' }: EmailFormProps) {
```

Comment:
we can add readonly identifier for every react props to enforce the principle of immutability in React functional components. Also I think it is better to add the props vertically:
```
export function EmailForm({ 
   selectedVendors, 
   onClose, 
   defaultContent = ' ' 
}: Readonly<EmailFormProps>)
```